### PR TITLE
Magento collection

### PR DESCRIPTION
**Pull request by AlteredCoder**

### CODE REVIEW

File: parsers/s01-parse/crowdsecurity/magento-extension-logs.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Parser for the CrowdSec Magento Extension.
```
Comment (he2ss) [inline parsers/s01-parse/crowdsecurity/magento-extension-logs.md:1]: missing link for the extension

**Review by he2ss — APPROVED** [review]

File: collections/crowdsecurity/magento.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## OPNSense collection
```
Comment (buixor) [inline collections/crowdsecurity/magento.md:1]: Leftover OPNSense

**Review by buixor — CHANGES_REQUESTED** [review]
minor remarks + do we want to add CC stuffing scenario or similar before merging ?

File: collections/crowdsecurity/magento.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+parsers:
+  - crowdsecurity/syslog-logs
+  - crowdsecurity/magento-extension-logs
+  - crowdsecurity/dateparse-enrich
```
Comment (buixor) [inline collections/crowdsecurity/magento.yaml:4]: geoip enrich

**Review by buixor — CHANGES_REQUESTED** [review]
minor changes

**Review by buixor — APPROVED** [review]
lgtm

**Pull request merged**